scripts/test-data/custom-field-description-meta.tsv

```diff
@@ -0,0 +1,3 @@
+custom-field-description	name	notifiable-locations	notifiable-animal-disease-investigation-category	notifiable-animal-disease-confirmation-category	start-date	end-date
```

Comment:
Am I being silly, I don't see where this file and the `custom-field-descriptions-meta.rsf` file are used?